src/test/scala/nl/knaw/dans/easy/licenses/LicensesJsonSpec.scala

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (C) 2019 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.licenses
+
+import java.io.File
+
+import org.json4s._
+import org.json4s.native.JsonMethods._
+import org.scalatest.{ FlatSpec, Matchers }
+
+class LicensesJsonSpec extends FlatSpec with Matchers {
+
+  val LICENSES_DIR = "src/main/assembly/dist/licenses"
+  val files = new File(LICENSES_DIR).listFiles.filter(_.isFile).map(_.getName).filter(n => !n.endsWith("properties") && !n.endsWith("json")).toList
+  val fileNames = files.map(f => f.substring(0, f.lastIndexOf(".")))
+  val json = parse(new File(LICENSES_DIR, "licenses.json"))
+  val viewNames = for {
+    JObject(file) <- json
+    JField("title", viewName) <- file
+  } yield viewName.values
+
+  "all the files in licenses.json" should "be present in the licenses directory" in {
+    viewNames.foreach(fileNames should contain(_))
```

Comment:
In your tests (like this one) you use `foreach`. While this works perfectly, it will fail the test on the first instance in `viewNames` it finds for which the expectation fails. If you look at

https://github.com/DANS-KNAW/easy-deposit-api/blob/80861017ade01760382fe7a643242682b26824d4/src/test/scala/nl.knaw.dans.easy.deposit/servlets/UploadSpec.scala#L58-L60

you'll see `forEvery` being used. This method collects all failing expectations and reports all of them at once, giving you a better overview of what errors occur.
For this method to work, you need to add `with Inspectors` to the class definition.